artview/components/plot_radar.py

```diff
@@ -466,6 +466,30 @@ def toolDefaultCmd(self):
                                           self.scan_type, self.rhi)
         self._update_plot()
 
+    def getPathInteriorValues(self, path):
+        '''
+        Return the bins values path.
+
+        Parameters
+        ----------
+        path - Matplotlib Path instance
+
+        Returns
+        -------
+        x, y, azi, range, value, ray_idx, range_inx: ndarray
+            Truplet of 1arrays containing x,y coordinate, azimuth,
+            range, current field value, ray index and range index
+            for all bin of the current radar and tilt inside path
+        '''
+        from .tools import interior
+        radar = self.Vradar.value
+        xy, idx = interior(path, self.Vradar.value, self.Vtilt.value)
+        aux = (xy[:, 0], xy[:, 1], radar.azimuth['data'][idx[:, 0]],
+               radar.range['data'][idx[:, 1]] / 1000.,
+               radar.fields[self.Vfield.value]['data'][idx[:, 0], idx[:, 1]],
+               idx[:, 0], idx[:, 1])
+        return aux
+
```

Comment:
The base here is that ROI is about space, not the data that is in that space, so it should not have access to Vradar. As a consequence Display must provide this information.